src/common/line_entry.rs

```diff
@@ -72,6 +72,56 @@ impl LineEntry {
         }
         comment::parse(self.raw_string.as_str())
     }
+
+    #[allow(dead_code)]
+    pub fn get_substitution_key(&self) -> Vec<&str> {
+        let mut keys = Vec::new();
+
+        let mut value = match self.get_value() {
+            Some(value) => value.trim(),
+            None => return keys,
+        };
+
+        if value.starts_with('\'') {
+            return keys;
+        }
+
+        fn escaped(prefix: &str) -> bool {
+            prefix.chars().rev().take_while(|ch| *ch == '\\').count() % 2 == 1
+        }
```

Comment:
Personally, I think closure seems more suitable for such cases 🙂 
```suggestion
        let is_escaped =
            |prefix: &str| prefix.chars().rev().take_while(|ch| *ch == '\\').count() % 2 == 1;
```

Reply:
Done